java/micronaut/config.yaml

```diff
@@ -0,0 +1,16 @@
+framework:
+  website: micronaut.io
+  version: 1.2.5
+
+image: gradle
```

Comment:
should be 
```yaml
deps:
  - gradle
```

I have made a slight modification to code base so as to run `java` frameworks on `alpine`

Reply:
so i just need to put as :
```
image: gradle
deps:
  - gradle
```